patches/chrome-BUILD.gn.patch

```diff
@@ -49,43 +49,44 @@ index a512d736c1d603680cc0e5bfb42afbd70a799f9a..0253e7271eaec572d76ce95185cb6fa5
      }
    }
  
-@@ -640,9 +644,11 @@ if (is_win) {
+@@ -640,9 +644,12 @@ if (is_win) {
        "--scm=0",
      ]
    }
 +  copy("brave_helper_plist") { deps = [ "//brave:brave_helper_plist" ] sources = [ "$root_gen_dir/brave/brave_helper_plist_tweaked.plist" ] outputs = [ "$root_gen_dir/chrome/{{source_file_part}}" ] }
  
    compile_entitlements("entitlements") {
      entitlements_templates = [ "app/app-entitlements.plist" ]
-+    if (is_official_build) { entitlements_templates += [ "app/app-entitlements-chrome.plist", "//brave/app/app-entitlements-brave.plist" ] }
++    import("//brave/app/entitlements.gni") entitlements_templates += brave_entitlements_templates
++    if (is_official_build) { entitlements_templates += [ "app/app-entitlements-chrome.plist" ] }
```

Comment:
we can fill both values in ```app/entitlements.gni``` and have only a one line patch here.

Reply:
Oops! I thought `if (is_official_build) { entitlements_templates += [ "app/app-entitlements-chrome.plist" ] }` is upsteam code

Reply:
Fixed.